lib/filter-stream.js

```diff
@@ -11,6 +11,7 @@ var BlockMap = require( './blockmap' )
  * @param {BlockMap} blockMap - the block map
  * @param {Object} [options] - options
  * @param {Boolean} [options.verify=true] - verify range checksums
+ * @param {Boolean} [options.calculateChecksums=false] - calculate range checksums and update blockmap
```

Comment:
How do `options.verify` and `options.calculateChecksums` interact? Is it expected that `verify` will always be used with blockmaps that **do** have checksums, and `calculateChecksums` will always be used with blockmaps that **don't** have checksums?
What happens if `options.verify` and `options.calculateChecksums` are both set to true at the same time? (should there be a unit-test for that?)

Reply:
This wouldn't make sense.
`verify` and `calculateChecksums` are exclusive.

Reply:
Should we rename `calculateChecksums` to `updateChecksums` or `generateChecksums`?

Reply:
changed to `generateChecksums`